### ISSUE TITLE
Controller - Typing text beyond the fold does not scroll

### ISSUE THREAD

**Issue by Ai-Kiwi** [visual-space/visual-editor#245]
For some reason scrolling doesn't happen when the lines go over the page and I have scrollable enabled. Is there another setting that I need to enable or is this a bug?

**Comment by adrian-moisa** [comment]
I checked the demo pages. The scroll works fine. Maybe there's something in your setup. I'd recommend to have a look in the demo pages. 
<img width="399" alt="image" src="https://github.com/visual-space/visual-editor/assets/11160141/7c1ab4e4-69c8-4f94-9d50-db1044ea9d46">


Btw, scrollable is enabled by default.
<img width="258" alt="image" src="https://github.com/visual-space/visual-editor/assets/11160141/349b18d2-8aa5-42dc-9721-51468740571e">

I'll wait for a reply from you to see if you managed to progress or if it's an actual issue. Then I'll close the ticket. Cheers!

**Comment by liquidiert** [comment]
Hey @adrian-moisa I have a similiar issue. Scrolling is activated but the component does not scroll down when text goes over the line. Here's a short video showing the issue:

https://github.com/visual-space/visual-editor/assets/36642878/efd19c12-0889-4c01-9391-bc1284c68f8c

So maybe you should reopen the issue?

**Comment by adrian-moisa** [comment]
Right, indeed, this is still an issue. Thanks for the video. There's a bit of broken logic. I'll have to dig in. Currently a bit busy, possibly end of month.

**Comment by liquidiert** [comment]
@adrian-moisa any news on this? If I can help somehow just let me know :)

**Comment by adrian-moisa** [comment]
If you can have a look, help will be much appreciated. I'm stuck in other duties this month and the next. Cheers!
